scst-store/release_notes.md

```diff
@@ -2,6 +2,45 @@
 
 This topic contains release notes for Supply Chain Security Tools - Store.
 
+## v1.0.0-beta.2
+
+**Release Date**: November 26, 2021
```

Comment:
The release date is actually when TAP beta 4 is released altogether. That's when the public can see it.

Anyways you can probably remove this.

Reply:
okay, removed